src/main/scala/org/broadinstitute/dsde/firecloud/service/WorkspaceService.scala

```diff
@@ -0,0 +1,64 @@
+package org.broadinstitute.dsde.firecloud.service
+
+import akka.actor.{Actor, Props}
+import com.wordnik.swagger.annotations._
+import org.broadinstitute.dsde.firecloud.WorkspaceClient
+import org.broadinstitute.dsde.firecloud.model.{WorkspaceEntity, WorkspaceIngest}
+import org.broadinstitute.dsde.firecloud.model.ModelJsonProtocol._
+import org.slf4j.LoggerFactory
+import spray.http.StatusCodes._
+import spray.httpx.SprayJsonSupport._
+import spray.routing._
+import org.broadinstitute.dsde.vault.common.directives.OpenAMDirectives._
+
+class WorkspaceServiceActor extends Actor with WorkspaceService {
+  def actorRefFactory = context
+  def receive = runRoute(createRoute)
+}
+
+@Api(value = "/workspaces", description = "Workspace Services", produces = "application/json")
+trait WorkspaceService extends HttpService with FireCloudDirectives {
+
+  private final val ApiPrefix = "workspaces"
+  private implicit val executionContext = actorRefFactory.dispatcher
+
+  val createRoute = createWorkspaceRoute
```

Comment:
for each service class, let's create a routes variable, and run all tests against all routes. This avoids test problems due to incorrect ordering of routes. See https://github.com/broadinstitute/vault-datamanagement/commit/07bbc6acd04e172766e6e441cc64ac11a5c3f9d7.